template-parts/flexible-cpts/date-filter.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+$filter_name = $args['name'];
+$filter_label = $args['label'];
+
+if(!empty($filter_label)){
+    $from_date_label = $filter_label . " (from)";
+    $to_date_label = $filter_label . " (to)";
+}
+else {
+    $from_date_label = "Date from";
+    $to_date_label = "Date to";
+}
+
+$from_date_name = $args['name'] . "_from_date";
+$to_date_name = $args['name'] . "_to_date";
+
+$from_date = get_query_var($from_date_name);
+$to_date = get_query_var($to_date_name);
+
+?>
+
+<div class="moj-datepicker" data-module="moj-date-picker">
+    <div class="govuk-form-group">
```

Comment:
```suggestion
    <div class="govuk-form-group govuk-!-margin-bottom-2">
```

Suggesting this because the gap beneath the first date picker and the second is larger than the gap between other filters.